tests/config/test_data.config

```diff
@@ -1,70 +1,71 @@
-// Base directory for test data
-def test_data_dir = "${launchDir}/tests/data/"
-
```

Comment:
Oops what happened here? Can we revert this change if the file hasn't changed?

Reply:
Ah, you added the bedgraph file. Wonder why it is not just showing that change. Did you copy and paste the entire file contents?

Reply:
Hi @drpatelh, I got conflicts merging the GoekeLab/module-master branch to nf-core/module-master branch, and I resolved it on the github resolve conflict page:
![Screenshot 2021-03-26 at 8 59 54 AM](https://user-images.githubusercontent.com/41866052/112562287-f28f5500-8e11-11eb-9baa-0e5ceaaa9a9f.png)